build/makelib/golang.mk

```diff
@@ -176,7 +176,7 @@ go.vet:
 
 .PHONY: go.fmt
 go.fmt:
-	@$(GOHOST) fmt $(GO_ALL_PACKAGES)
+	@gofmt_out=$$(gofmt -d -e $(GO_SUBDIRS) 2>&1) && [ -z "$${gofmt_out}" ] || (echo "$${gofmt_out}" 1>&2; exit 1)
```

Comment:
I've always used curly braces to group commands in a pipeline like around the echo and exit group. Not sure what the exact difference between that and the sub shell parens you're using.